src/app/page.tsx

```diff
@@ -2,7 +2,7 @@ import Image from "next/image";
 
 export default function Home() {
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
+    <main className="min-h-screen flex-col items-center justify-between p-24">
```

Comment:
Removing the `flex` class might break some existing layout functionality, especially if other elements rely on the flexbox container. Ensure that the layout still works as intended.